Security/LoginManager.php

```diff
@@ -27,16 +28,19 @@
  */
 class LoginManager implements LoginManagerInterface
 {
-    private $securityContext;
+    /**
+     * @var SecurityContextInterface|TokenStorageInterface
+     */
+    private $tokenStorage;
     private $userChecker;
     private $sessionStrategy;
     private $container;
 
-    public function __construct(SecurityContextInterface $context, UserCheckerInterface $userChecker,
+    public function __construct($tokenStorage, UserCheckerInterface $userChecker,
                                 SessionAuthenticationStrategyInterface $sessionStrategy,
                                 ContainerInterface $container)
     {
```

Comment:
you should add a check in the controller to accept only these 2 interfaces (see how Symfony 2.6 does it)